### PR TITLE
Updating ose-machine-config-operator builder & base images to be consistent with ART

### PR DESCRIPTION
**Pull request by jupierce**
Updating ose-machine-config-operator builder & base images to be consistent with ART
Reconciling with https://github.com/openshift/ocp-build-data/tree/f66c03011773dc3755ad874fc691be612914d65f/images/ose-machine-config-operator.yml

If you have any questions about this pull request, please reach out to `@art-team` in the `#aos-art` coreos slack channel.

Depends on https://github.com/openshift/images/pull/44 . Allow it to merge and then run `/test all` on this PR.

### CODE REVIEW

**Review by ashcrow — APPROVED** [review]

**Comment by kikisdeliveryservice** [comment]
so holding lgtm until openshift/images#44  merges

/hold

**Pull request merged**
